src/workloads/scale/LLTMixed.yml

```diff
@@ -241,56 +241,81 @@ Actors:
         LogEvery: 15 minutes
         Blocking: None
 
-- Name: MixedBaseline.Low
+- Name: Low.Short.Insert
```

Comment:
these names are different to the other workloads.

That is Low.Short.Insert v InsertBenchmark.Low.Medium.